### DISCUSSION TITLE
I ask a question on Keycloak Discourse and my account become pending.

### DISCUSSION THREAD

**Discussion by SetoKaiba** [keycloak/keycloak#17005] [Q&A] [answered]
I ask a question on Keycloak Discourse and my account become pending. No one reviews my post, and I can't post anymore. Can anyone helps?

**Comment by antonfil** [comment]
Me too )

**Comment by SetoKaiba** [comment, marked as answer]
Can anyone help?

**UPDATE:** Solved. I'm approved.
